astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -634,10 +634,12 @@ def test_all(self):
         with pytest.raises(TypeError):
             np.all(self.q)
 
+    @pytest.mark.filterwarnings("ignore:`sometrue` is deprecated as of NumPy 1.25.0")
     def test_sometrue(self):
         with pytest.raises(TypeError):
             np.sometrue(self.q)
```

Comment:
Huh, so how are we supposed to test this when `np.sometrue` is completely removed? Any idea, @mhvk ?

Reply:
Can't we just put this in a `if NUMPY_LT_XX` block when version XX removes `np.sometrue`?  `np.sometrue` is the same as `np.any`.

Reply:
Maybe we should do it like in #14484, where we ended up just using a version-dependent test; see https://github.com/astropy/astropy/blob/36e30a75d3ca704991d6d60d264476022d5b488d/astropy/units/tests/test_quantity_non_ufuncs.py#L670-L677